### ISSUE TITLE
Issues in systemd man pages

### ISSUE THREAD

**Issue by hmartink** [systemd/systemd#35307]
### systemd version the issue has been seen with

256.7

### Used distribution

Several

### CPU architectures issue was seen on

None

### Expected behaviour you didn't see

Not the issues in the next box

### Unexpected behaviour you saw

`Dear systemd maintainer,
the manpage-l10n project[1] maintains a large number of translations of
man pages both from a large variety of sources (including systemd) as
well for a large variety of target languages.

During their work translators notice different possible issues in the
original (english) man pages. Sometimes this is a straightforward
typo, sometimes a hard to read sentence, sometimes this is a
convention not held up and sometimes we simply do not understand the
original.

We use several distributions as sources and update regularly (at
least every 2 month). This means we are fairly recent (some
distributions like archlinux also update frequently) but might miss
your latest upstream version once in a while, so the error might be
already fixed. We apologize and ask you to close the issue immediately
if this should be the case, but given the huge volume of projects and
the very limited number of volunteers we are not able to double check
each and every issue.

Secondly we translators see the manpages in the neutral po format,
i.e. converted and harmonized, but not the original source (be it man,
groff, xml or other). So we cannot provide a true patch (where
possible), but only an approximation which you need to convert into
your source format.

Finally the issues I'm reporting have accumulated over time and are
not always discovered by me, so sometimes my description of the
problem my be a bit limited - do not hesitate to ask so we can clarify
them.

I'm now reporting the issues for your project. If future reports
should use another channel, please let me know. (These forms are very (!) inconvenient and error prone to use!)

[1] https://manpages-l10n-team.pages.debian.net/manpages-l10n/

First, should mkosi bugs be reported here as well or elsewhere?

For the rest, see the attachement:

[manbugreport.systemd.txt](https://github.com/user-attachments/files/17873907/manbugreport.systemd.txt)

### Steps to reproduce the problem

Call the respective man page

**Comment by yuwata** [comment]
- [ ] Man page: capsule@.service.5
Issue:    \"p-\" → \"p_\" ?
```
"Capsule names may be chosen freely by the user, however, they must be "
"suitable as UNIX filenames (i\\&.e\\&. 255 characters max, and contain no \"/"
"\"), and when prefixed with \"p-\" be suitable as a user name matching "
"strict POSIX rules, see \\m[blue]B<User/Group Name "
"Syntax>\\m[]\\&\\s-2\\u[1]\\d\\s+2 for details\\&."
```

I do not think so??

- [x] Man page: file-hierarchy.7
Issue:    have provide → have provided
```
"Note that resources placed in this directory typically are under shared "
"ownership, i\\&.e\\&. multiple different packages have provide and consume "
"these resources, on equal footing, without any obvious primary owner\\&. "
"This makes makes things systematically different from /usr/lib/, where "
"ownership is generally not shared\\&."
```
- [x] Man page: homectl.1
Issue:    suitible → suitable
```
"Takes a comma- or colon-separated list of languages preferred by the user, "
"ordered by descending priority\\&. The I<$LANG> and I<$LANGUAGE> environment "
"variables are initialized from this value on login, and thus values suitible "
"for these environment variables are accepted here, for example B<--"
"language=de_DE\\&.UTF-8>\\&. This option may be used more than once, in "
"which case the language lists are concatenated\\&."
```
- [ ] Man page: homectl.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
There exists citations to less below, not necessary.
Also, systemd(1) is for the service manager, not suitable here.

- [x] Man page: importctl.1
Issue:    are one → in one
```
"B<importctl> operates both on block-level disk images (such as DDIs) as well "
"as file-system-level images (tarballs)\\&. It supports disk images are one "
"of the four following classes:"
```
- [x] Man page: importctl.1
Issue:    an managed → and managed
```
"Portable service images, that may be attached an managed via "
"B<portablectl>(1)\\&."
```
- [ ] Man page: importctl.1
Issue:    "etag" is not explained, what is this? Is this related to ETag as of https://en.wikipedia.org/wiki/HTTP_ETag
```
"If B<-keep-download=yes> is specified the image will be downloaded and "
"stored in a read-only subvolume/directory in the image directory that is "
"named after the specified URL and its HTTP etag\\&. A writable snapshot is "
"then taken from this subvolume, and named after the specified local name\\&. "
"This behavior ensures that creating multiple instances of the same URL is "
"efficient, as multiple downloads are not necessary\\&. In order to create "
"only the read-only image, and avoid creating its writable snapshot, specify "
"\"-\" as local name\\&."
```
- [x] Man page: importctl.1
Issue:    Is C-c → Control-C? If so, plese spell it out.
```
"Note that pressing C-c during execution of this command will not abort the "
"download\\&. Use B<cancel-transfer>, described below\\&."
```
- [x] Man page: importctl.1
Issue:    Here it is "\\&.qcow2 format" format, further down "qcow2" - unify?
```
"If the downloaded image is in \\&.qcow2 format it is converted into a raw "
"image file before it is made available\\&."
```
- [ ] Man page: importctl.1
Issue 1:  tar archive → B<tar>(1) archive
Issue 2:  xz → B<xz>(1)
Issue 3:  gzip → B<gzip>(1)
Issue 4:  bzip2 → B<bzip2>(1)
```
"Imports a TAR or RAW image, and places it under the specified name in the "
"image directory for the image class selected via B<--class=>\\&. When "
"B<import-tar> is used, the file specified as the first argument should be a "
"tar archive, possibly compressed with xz, gzip or bzip2\\&. It will then be "
"unpacked into its own subvolume/directory\\&. When B<import-raw> is used, "
"the file should be a qcow2 or raw disk image, possibly compressed with xz, "
"gzip or bzip2\\&. If the second argument (the resulting image name) is not "
"specified, it is automatically derived from the file name\\&. If the "
"filename is passed as \"-\", the image is read from standard input, in which "
"case the second argument is mandatory\\&."
```
- [ ] Man page: importctl.1
Issue:    btrfs → B<btrfs>(5)
```
"Imports an image stored in a local directory into the image directory for "
"the image class selected via B<--class=> and operates similarly to B<import-"
"tar> or B<import-raw>, but the first argument is the source directory\\&. If "
"supported, this command will create a btrfs snapshot or subvolume for the "
"new image\\&."
```
- [ ] Man page: importctl.1
Issue 1:  gzip → B<gzip>(1)
Issue 2:  xz → B<xz>(1)
Issue 3:  bzip2 → B<bzip2>(1)

"Exports a TAR or RAW image and stores it in the specified file\\&. The first "
"parameter should be an image name\\&. The second parameter should be a file "
"path the TAR or RAW image is written to\\&. If the path ends in \"\\&.gz\", "
"the file is compressed with gzip, if it ends in \"\\&.xz\", with xz, and if "
"it ends in \"\\&.bz2\", with bzip2\\&. If the path ends in neither, the file "
"is left uncompressed\\&. If the second argument is missing, the image is "
"written to standard output\\&. The compression may also be explicitly "
"selected with the B<--format=> switch\\&. This is in particular useful if "
"the second parameter is left unspecified\\&."
msgstr ""
"Exportiert ein TAR- oder RAW-Abbild, und speichert es in der angegebenen "
"Datei\\&. Der erste Parameter sollte ein Abbildname sein\\&. Der zweite "
"Parameter sollte ein Dateipfad sein, in den das TAR- oder RAW-Abbild "
"geschrieben wird\\&. Falls der Pfad auf »\\&.gz« endet, wird die Datei mit "
"B<gzip>(1) komprimiert, falls er auf »\\&.xz« endet, mit B<xz>(1) und falls "
"er auf »\\&.bz2« endet, mit B<bzip2>(1)\\&. Falls der Pfad auf keine dieser "
"Zeichenketten endet, verbleibt die Datei unkomprimiert\\&. Falls das zweite "
"Argument fehlt, wird das Abbild in die Standardausgabe geschrieben\\&. Die "
"Komprimierung kann auch explizit mit dem Schalter B<--format=> ausgewählt "
"werden\\&. Dies ist insbesondere nützlich, falls der zweite Parameter nicht "
"angegeben wird\\&."
```
- [ ] Man page: importctl.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
No.
- [ ] Man page: journalctl.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
No.

- [ ] Man page: journald.conf.5
Issue:    B<systemd-journal-remote> → B<systemd-journal-remote>(8)
```
"When forwarding over a socket the \\m[blue]B<Journal Export "
"Format>\\m[]\\&\\s-2\\u[4]\\d\\s+2 is used when sending over the wire\\&. "
"Notably this includes the metadata field I<__REALTIME_TIMESTAMP> so that "
"B<systemd-journal-remote> (see B<systemd-journal-remote.service>(8)) can be "
"used to receive the forwarded journal entries\\&."
```
The page has many links to the page, I do not think it is necessary to add more.
- [x] Man page: kernel-install.8
Issue:    devicetree → device tree   (as in previous paragraph?)
```
"The devicetree file contains a path, and this path specifies a location "
"relative to the kernel install tree\\&. A set of locations is checked, "
"including in particular /usr/lib/modules/I<KERNEL_VERSION>/dtb/, which is "
"the recommended location to place the dtb files under\\&. For example, with "
"\"broadcom/bcm2711-rpi-4-b\\&.dtb\" in the devicetree file, the device tree "
"blob for the Raspberry Pi 4 Model B would be installed, and the actual file "
"would be /usr/lib/modules/I<KERNEL_VERSION>/dtb/broadcom/bcm2711-rpi-4-b\\&."
"dtb\\&."
```
- [ ] Man page: localectl.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
No.
- [ ] Man page: loginctl.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
No.
- [ ] Man page: machinectl.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
No.
- [x] Man page: machinectl.1
Issue:    Is C-c → Control-C? If so, please spell it out.
```
"Note that pressing C-c during execution of this command will not abort the "
"download\\&. Use B<cancel-transfer>, described below\\&."
```
The sentence is already removed.
- [ ] Man page: networkctl.1
Issue:    B<systemd-networkd> → B<systemd-networkd>(8)
```
"B<systemd-networkd> is not handling the link\\&."

"B<systemd-networkd> failed to configure the link\\&."

"The link is gone, but has not yet been dropped by B<systemd-networkd>\\&."
```
I do not think it is necessary. It already has many links to that page.
- [x] Man page: networkctl.1
Issue 1:  systemd-networkd\\&.service → B<systemd-networkd\\&.service>(8)
Issue 2:  systemd-networkd-persistent-storage\\&.service → B<systemd-networkd-persistent-storage\\&.service>
```
"Notify systemd-networkd\\&.service that the persistent storage for the "
"service is ready\\&. This is called by systemd-networkd-persistent-"
"storage\\&.service\\&. Usually, this command should not be called manually "
"by users or administrators\\&."
```
- [x] Man page: pam_systemd.8
Issue:    systemd-user-sessions\\&.service → B<systemd-user-sessions\\&.service>(8)
```
"Similar to \"user\" but sessions of this class are not ordered after systemd-user-sessions\\&.service, i\\&.e\\&. may be started before regular sessions are allowed to be established\\&. This session class is the default for sessions of the root user that would otherwise qualify for the B<user> class, see above\\&. (Added in v256\\&.)"
```
- [ ] Man page: pam_systemd.8
Issue:    B<ssh> → B<ssh>(1)
```
"Similar to \"user\" but for sessions which are not fully set up yet, i\\&.e\\&. have no home directory mounted or similar\\&. This is used by B<systemd-homed.service>(8) to allow users to log in via B<ssh> before their home directory is mounted, delaying the mount until the user provided the unlock password\\&. Sessions of this class are upgraded to the regular B<user> class once the home directory is activated\\&."
```
- [ ] Man page: pam_systemd.8
Issue:    B<cron> → B<cron>(8)
```
"Used for background sessions, such as those invoked by B<cron> and similar tools\\&. This is the default class for sessions for which no TTY or X display is known at session registration time\\&."
```
- [x] Man page: pam_systemd.8
Issue 1:  In previous strings the class name was done in quotes, not in bold, should be consistent, e.g. B<background> → \"background\"
Issue 2:  user@\\&.service → B<user@\\&.service>(5)
```
"Similar to B<background>, but sessions of this class will not pull in the user@\\&.service of the user, and thus possibly have no services of the user running\\&. (Added in v256\\&.)"
```
- [x] Man page: pam_systemd.8
Issue:    user@\\&.service → B<user@\\&.service>(5)
```
"The user@\\&.service service of the user is registered under this session class\\&. (Added in v256\\&.)"
```
- [x] Man page: pam_systemd.8
Issue:    In previous strings the class name was done in quotes, not in bold, should be consistent, e.g. B<manager> → \"manager\"
```
msgid "Similar to B<manager>, but for the root user\\&. Compare with the B<user> vs\\&. B<user-early> situation\\&. (Added in v256\\&.)"
```
- [ ] Man page: pam_systemd_loadkey.8
Issue:    pam(7) does not use authtok nor token; spell it out?
```
"pam_systemd_loadkey - Read password from kernel keyring and set it as PAM "
"authtok"
```
???
- [x] Man page: pam_systemd_loadkey.8
Issue:    pam_systemd_loadkey → B<pam_systemd_loadkey>
```
"In this setup, early during the boot process, B<systemd-cryptsetup@."
"service>(8)  will ask for the passphrase and store it in the kernel keyring "
"with the keyname \"cryptsetup\"\\&. Then when the display manager does the "
"autologin, pam_systemd_loadkey will read the passphrase from the kernel "
"keyring, set it as the PAM authtok, and then pam_gnome_keyring and "
"pam_kwallet5 will unlock with the same passphrase\\&."
```
- [ ] Man page: portablectl.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
No.
- [ ] Man page: poweroff.8
Issue:    B<halt> → B<halt>(8)
```
"Note that on many SysV systems B<halt> used to be synonymous to B<poweroff>, "
"i\\&.e\\&. both commands would equally result in powering the machine "
"off\\&. systemd is more accurate here, and B<halt> results in halting the "
"machine only (leaving power on), and B<poweroff> is required to actually "
"power it off\\&."
```
No. halt(8) is an alias of poweroff(8), and we usually do not add self citation.
- [ ] Man page: repart.d.5
Issue:    btrfs → B<btrfs>(5)
```
"Note that this option only takes effect if the target filesystem supports "
"subvolumes, such as \"btrfs\"\\&."

"Specifies where and how the partition should be mounted\\&. Takes at least "
"one and at most two fields separated with a colon (\":\")\\&. The first "
"field specifies where the partition should be mounted\\&. The second field "
"specifies extra mount options to append to the default mount options\\&. "
"These fields correspond to the second and fourth column of the B<fstab>(5)  "
"format\\&. This setting may be specified multiple times to mount the "
"partition multiple times\\&. This can be used to add mounts for different "
"btrfs subvolumes located on the same btrfs partition\\&."
```
- [ ] Man page: repart.d.5
Issue:    mkfs\\&.btrfs → B<mkfs.btrfs>(8)
```
"Note that due to limitations of \"mkfs\\&.btrfs\", this option is only "
"supported when running with B<--offline=no>\\&."
```
- [ ] Man page: repart.d.5
Issue:    systemd-repart → B<systemd-repart>(8)
```
"Configures the data block size of the generated verity hash partition\\&. "
"Must be between 512 and 4096 bytes and must be a power of 2\\&. Defaults to "
"the sector size if configured explicitly, or the underlying block device "
"sector size, or 4K if systemd-repart is not operating on a block device\\&."

"Configures the hash block size of the generated verity hash partition\\&. "
"Must be between 512 and 4096 bytes and must be a power of 2\\&. Defaults to "
"the sector size if configured explicitly, or the underlying block device "
"sector size, or 4K if systemd-repart is not operating on a block device\\&."
```
No, the page already has many citation to the page.
- [x] Man page: repart.d.5
Issue:    Specify → Specifies
```
"Specify how the encrypted partition should be set up\\&. Takes at least one "
"and at most three fields separated with a colon (\":\")\\&. The first field "
"specifies the encrypted volume name under /dev/mapper/\\&. If not specified, "
"\"luks-UUID\" will be used where \"UUID\" is the LUKS UUID\\&. The second "
"field specifies the keyfile to use following the same format as specified in "
"crypttab\\&. The third field specifies a comma-delimited list of crypttab "
"options\\&. These fields correspond to the first, third and fourth column of "
"the B<crypttab>(5)  format\\&."
```
- [x] Man page: resolvectl.1
Issue:    Either "single-label" or "single label" - should be consistent
```
"Takes a boolean parameter; used in conjunction with B<query>\\&. If true, "
"rules regarding routing of single-label names are relaxed\\&. Defaults to "
"false\\&. By default, lookups of single label names are assumed to refer to "
"local hosts to be resolved via local resolution such as LLMNR or via search "
"domain qualification and are not routed to upstream servers as is\\&. If "
"this option is enabled these rules are disabled and the queries are routed "
"upstream anyway\\&. Also see the I<ResolveUnicastSingleLabel=> option in "
"B<resolved.conf>(5)  which provides a system-wide option that controls this "
"behaviour\\&."
```
- [ ] Man page: resolved.conf.5
Issue:    B<systemd-resolved> → B<systemd-resolved>(8)
```
"B<systemd-resolved> always attempts to reach the upstream DNS servers first, "
"before providing the client application with any stale data\\&. If this "
"feature is enabled, cache will not be flushed when changing servers\\&."
```
The page already contains the citation. I do not think it is necessary.
- [ ] Man page: run0.1
Issue:    What is "multi-call invocation"?
```
"Note that B<run0> is implemented as an alternative multi-call invocation of "
"B<systemd-run>(1)\\&."
```
- [x] Man page: run0.1
Issue:    on the service → of the service
```
"Sets a property on the service unit that is created\\&. This option takes an "
"assignment in the same format as B<systemctl>(1)\\*(Aqs B<set-property> "
"command\\&."
```
- [x] Man page: run0.1
Issue:    on a → in a
```
"Execute operation on a local container\\&. Specify a container name to "
"connect to\\&."
```
- [ ] Man page: run0.1
Issue:    B<systemd-run> → B<systemd-run>(1)
```
"As with B<systemd-run>, the session will inherit the system environment from "
"the service manager\\&. In addition, the following environment variables "
"will be set:"
```
No.
- [ ] Man page: systemctl.1
Issue:    First sentence broken? can still be → is
```
"If a unit that gets stopped can still be triggered by other units, a warning "
"containing the names of the triggering units is shown\\&.  B<--no-warn> can "
"be used to suppress the warning\\&."
```
I do not think so.
- [x] Man page: systemctl.1
Issue:    is forked → which is forked
```
"Note that this shows the I<effective> block, i\\&.e\\&. the combination of "
"environment variables configured via configuration files, environment "
"generators and via IPC (i\\&.e\\&. via the B<set-environment> described "
"below)\\&. At the moment a unit process is forked off this combined "
"environment block will be further combined with per-unit environment "
"variables, which are not visible in this command\\&."
```
- [ ] Man page: systemctl.1
Issue:    B<systemd-logind> → B<systemd-logind>(8)
```
"If B<--force> is specified, and B<systemd-logind> returned error for the "
"operation, the error will be ignored and the operation will be tried again "
"directly through starting the target unit\\&."

"When used with B<suspend>, B<hibernate>, B<hybrid-sleep>, or B<suspend-then-"
"hibernate>, the error returned by B<systemd-logind> will be ignored, and the "
"operation will be performed directly through starting the corresponding "
"units\\&."
```
Not necessary.
- [ ] Man page: systemctl.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
No.
- [ ] Man page: systemd.1
Issue:    B<sd_notify()> → B<sd_notify>(3)
```
"The service listens to various UNIX process signals that can be used to "
"request various actions asynchronously\\&. The signal handling is enabled "
"very early during boot, before any further processes are invoked\\&. "
"However, a supervising container manager or similar that intends to request "
"these operations via this mechanism must take into consideration that this "
"functionality is not available during the earliest initialization phase\\&. "
"An B<sd_notify()> notification message carrying the "
"I<X_SYSTEMD_SIGNALS_LEVEL=2> field is emitted once the signal handlers are "
"enabled, see below\\&. This may be used to schedule submission of these "
"signals correctly\\&."
```
NAK.
- [ ] Man page: systemd.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
NAK.
- [ ] Man page: systemd.1
Issue:    consumed → used    This is for sending, not reading?
```
"Set by service manager for its services for status and readiness "
"notifications\\&. Also consumed by service manager for notifying supervising "
"container managers or service managers up the stack about its own "
"progress\\&. See B<sd_notify>(3)  and the relevant section below for more "
"information\\&."
```
NAK.
- [x] Man page: systemd.1
Issue:    indication → indication of
```
"Signals sent to PID 1 before this message is sent might not be handled "
"correctly yet\\&. A consumer of these messages should parse the value as an "
"unsigned integer indication the level of support\\&. For now only the "
"mentioned level 2 is defined, but later on additional levels might be "
"defined with higher integers, that will implement a superset of the "
"currently defined behaviour\\&."
```
- [x] Man page: systemd.1
Issue:    systemd → B<systemd>
```
"Specify what to do when the system manager (PID 1) crashes\\&. This switch "
"has no effect when systemd is running as user instance\\&. See I<systemd\\&."
"crash_action=> above\\&."
```
- [ ] Man page: systemd-analyze.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
msgstr ""
"Beachten Sie, dass das Setzen der regulären Umgebungsvariablen "
"I<$LESSCHARSET> keine Auswirkungen auf die Ausführungen von B<less>(1) durch "
"B<systemd>(1)-Werkzeuge hat\\&."
```
NAK
- [x] Man page: systemd-backlight@.service.8
Issue:    systemd-backlight@\\&.service → B<systemd-backlight@\\&.service>
```
"systemd-backlight@\\&.service is a service that restores the brightness of a "
"display backlight or LED (e\\&.g\\&. keyboard backlight) device at early "
"boot, and saves it at shutdown\\&. The brightness is stored in /var/lib/"
"systemd/backlight/\\&."
```
- [x] Man page: systemd-boot.7
Issue:    Is "option" really part of the quoted string?
```
"A \"Reboot Into Firmware Interface option\", if supported by the UEFI "
"firmware\\&."
```
- [x] Man page: systemd-bsod.service.8
Issue:    systemd-bsod\\&.service → B<systemd-bsod\\&.service>
```
"systemd-bsod\\&.service is used to display a blue screen which contains a "
"message relating to a boot failure, including a QR code which can be scanned "
"to get helpful information about the failure\\&."
```
- [x] Man page: systemd-cryptenroll.1
Issue:    password/passhprase → password/passphrase
```
"Use a TPM2 device instead of a password/passhprase read from stdin to unlock "
"the volume\\&. Expects a device node path referring to the TPM2 chip (e\\&."
"g\\&.  /dev/tpmrm0)\\&. Alternatively the special value \"auto\" may be "
"specified, in order to automatically determine the device node of a "
"currently discovered TPM2 device (of which there must be exactly one)\\&."
msgstr ""
"Ein TPM2-Gerät anstelle eines aus der Standardeingabe gelesenen Passwortes/"
"einer Passphrase zum Entsperren des Datenträgers verwenden\\&. Erwartet ein "
"Geräteknotepnpfad, der sich auf einen TPM2-Chip bezieht (z\\&. B\\&. /dev/"
"tpmrm0)\\&. Alternativ kann der besondere Wert »auto« angegeben werden, um "
"automatisch den Geräteknoten eines aktuell erkannten TPM2-Gerätes zu "
"bestimmen (davon darf es nur genau einen geben)\\&."
```
- [x] Man page: systemd-cryptsetup.8
Issue 1:  systemd-cryptsetup → B<systemd-cryptsetup>
Issue 2:  systemd-cryptsetup@\\&.service → B<systemd-cryptsetup@\\&.service>(8)
Issue 3:  In SYNPOSIS SOURCE-DEVICE, here SOURCEDEVICE
Issue 4:  CRYPTTAB-OPTIONS does not appear in SYNPOSIS, but CONFIG is used there
```
"systemd-cryptsetup is used to set up (with B<attach>) and tear down (with "
"B<detach>) access to an encrypted block device\\&. It is primarily used via "
"systemd-cryptsetup@\\&.service during early boot, but may also be called "
"manually\\&. The positional arguments I<VOLUME>, I<SOURCE-DEVICE>, I<KEY-"
"FILE>, and I<CRYPTTAB-OPTIONS> have the same meaning as the fields in "
"B<crypttab>(5)\\&."
```
- [x] Man page: systemd-cryptsetup.8
Issue:    systemd-cryptsetup@\\&.service → B<systemd-cryptsetup@\\&.service>(8)
```
"systemd-cryptsetup@\\&.service is a service responsible for providing access "
"to encrypted block devices\\&. It is instantiated for each device that "
"requires decryption\\&."

"systemd-cryptsetup@\\&.service instances are part of the system-"
"systemd\\ex2dcryptsetup\\&.slice slice, which is destroyed only very late in "
"the shutdown procedure\\&. This allows the encrypted devices to remain up "
"until filesystems have been unmounted\\&."

"In order to unlock a volume a password or binary key is required\\&.  "
"systemd-cryptsetup@\\&.service tries to acquire a suitable password or "
"binary key via the following mechanisms, tried in order:"
```

- [x] Man page: systemd-cryptsetup.8
Issue:    Why PIN and not pin as above?
```
"This credential specifies the PIN requested by generic LUKS2 token "
"modules\\&."
```
- [x] Man page: systemd-firstboot.1
Issue:    systemd-firstboot\\&.service → B<systemd-firstboot\\&.service>
```
"The B<systemd-firstboot\\&.service> unit is one of the units which are used "
"to initialize the machine configuration during \"First Boot\", i\\&.e\\&. "
"when the system is freshly installed or after a factory reset\\&. The "
"B<systemd>(1)  manager itself will initialize B<machine-id>(5)  and preset "
"all units, enabling or disabling them according to the B<systemd.preset>(5)  "
"settings\\&.  systemd-firstboot\\&.service is started later to interactively "
"initialize basic system configuration\\&. It is started only if "
"I<ConditionFirstBoot=yes> is met, which essentially means that /etc/ is "
"unpopulated, see B<systemd.unit>(5)  for details\\&. System credentials may "
"be used to inject configuration; those settings are not queried "
"interactively\\&."
```
- [ ] Man page: systemd-fsck@.service.8
Issue:    B<fsck> → B<fsck>(8)
```
"systemd-fsck-root\\&.service and systemd-fsck-usr\\&.service will activate "
"reboot\\&.target if B<fsck> returns the \"System should reboot\" condition, "
"or emergency\\&.target if B<fsck> returns the \"Filesystem errors left "
"uncorrected\" condition\\&."

"systemd-fsck@\\&.service will fail if B<fsck> returns with either \"System "
"should reboot\" or \"Filesystem errors left uncorrected\" conditions\\&. For "
"filesystems listed in /etc/fstab without \"nofail\" or \"noauto\" options, "
"\"local-fs\\&.target\" will then activate emergency\\&.target\\&."
```
- [x] Man page: systemd-hibernate-resume.service.8
Issue:    B<systemd-hibernate-resume> → B<systemd-hibernate-resume>(8)
```
"B<systemd-hibernate-resume> only supports the in-kernel hibernation "
"implementation, see \\m[blue]B<Swap suspend>\\m[]\\&\\s-2\\u[1]\\d\\s+2\\&. "
"Internally, it works by writing the major:minor of selected device node to /"
"sys/power/resume, along with the offset in memory pages (/sys/power/"
"resume_offset) if supported\\&."
"B<systemd-hibernate-resume>(8) unterstützt nur die kernelinterne "
"Ruhezustands-Implementierung, siehe "
"\\m[blue]B<Auslagerungssuspendierung>\\m[]\\&\\s-2\\u[1]\\d\\s+2\\&. Intern "
"funktioniert sie durch Schreiben des Major- und Minor-Wertes des "
"ausgewählten Geräteknotens nach /sys/power/resume, zusammen mit dem Versatz "
"in Speicherseiten (/sys/power/resume_offset), falls unterstützt\\&."
```
self citation, NAK
- [x] Man page: systemd-import-generator.8
Issue:    No I<> around?
```
"verify="
```
- [x] Man page: systemd-inhibit.1
Issue 1:  B<less> → B<less>(1)
Issue 2: systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
NAK.
- [ ] Man page: systemd-journald.service.8
Issue 1:  syslog → B<syslog>(2)
Issue 2:  wall → B<wall>(1)
```
"Controls the maximum log level of messages that are stored in the journal, "
"forwarded to syslog, kmsg, the console, the wall, or a socket\\&. This "
"kernel command line options override the settings of the same names in the "
"B<journald.conf>(5)  file\\&."
```
- [x] Man page: systemd.link.5
Issue:    systemd-networkd → B<systemd-networkd>(8)
```
"This setting is useful to configure the \"ID_NET_MANAGED_BY=\" property "
"which declares which network management service shall manage the interface, "
"which is respected by systemd-networkd and others\\&. Use"

"to declare explicitly that B<systemd-networkd> shall manage the interface, "
"or set the property to something else to declare explicitly it shall not do "
"so\\&. See B<systemd.network>(5)  for details how this property is used to "
"match interface names\\&."
```
- [x] Man page: systemd.link.5
Issue 1:  First sentence unclear. How can a list/indices forward traffic?
 -> Read the second sentence.
Issue 2:  \"all\" in which will include → \"all\", which includes
Issue 3: In the middle the term "affinity masks" is used. Is this the same as RPS?
```
"Configures Receive Packet Steering (RPS) list of CPUs to which RPS may "
"forward traffic\\&. Takes a list of CPU indices or ranges separated by "
"either whitespace or commas\\&. Alternatively, takes the special value "
"\"all\" in which will include all available CPUs in the mask\\&. CPU ranges "
"are specified by the lower and upper CPU indices separated by a dash (e\\&."
"g\\&.  \"2-6\")\\&. This option may be specified more than once, in which "
"case the specified CPU affinity masks are merged\\&. If an empty string is "
"assigned, the mask is reset, all assignments prior to this will have no "
"effect\\&. Defaults to unset and RPS CPU list is unchanged\\&. To disable "
"RPS when it was previously enabled, use the special value \"disable\"\\&."
```
- [ ] Man page: systemd-machined.service.8
Issue:    SSH → B<ssh>(1)
```
"The daemon provides both a C library interface (which is shared with "
"B<systemd-logind.service>(8)) as well as a D-Bus interface and a Varlink "
"interface\\&. The library interface may be used to introspect and watch the "
"state of virtual machines/containers\\&. The bus interface provides the same "
"but in addition may also be used to register or terminate machines\\&. The "
"Varlink interface may be used to register machines with optional extensions, "
"e\\&.g\\&. with an SSH key / address; it can be queried with B<varlinkctl "
"introspect /run/systemd/machine/io\\&.systemd\\&.Machine io\\&.systemd\\&."
"Machine>\\&. For more information please consult B<sd-login>(3)  and B<org."
"freedesktop.machine1>(5)  and B<org.freedesktop.LogControl1>(5)\\&."
```
- [x] Man page: systemd-machine-id-setup.1
Issue:    chroot → B<chroot>(8)
```
"If a valid machine ID is stored in /run/machine-id, the machine ID is copied "
"and used to initialize the machine ID in /etc/machine-id\\&. This step is "
"skipped if B<--root=> is specified or running in a chroot environment\\&."

"If a valid machine ID is provided through B<system\\&.machine_id> "
"credential, the machine ID is copied and used to initialize the machine ID "
"in /etc/machine-id\\&. This step is skipped if B<--root=> is specified or "
"running in a chroot environment\\&."

"If run inside a KVM virtual machine and a UUID is configured (via the B<-"
"uuid> option), this UUID is used to initialize the machine ID\\&. The caller "
"must ensure that the UUID passed is sufficiently unique and is different for "
"every booted instance of the VM\\&. This step is skipped if B<--root=> is "
"specified or running in a chroot environment\\&."
```
- [x] Man page: systemd.mount.5
Issue:    What does "If dosn't" in the last sentence relates to?
```
"Takes a boolean argument\\&. If true or no argument, a I<BindsTo=> "
"dependency on the backing device is set\\&. If false, the mount unit is not "
"stopped no matter whether the backing device is still present\\&. This is "
"useful when the file system is backed by volume managers\\&. If not set, and "
"the mount comes from unit fragments, i\\&.e\\&. generated from /etc/fstab by "
"B<systemd-fstab-generator>(8)  or loaded from a manually configured mount "
"unit, a combination of I<Requires=> and I<StopPropagatedFrom=> dependencies "
"is set on the backing device\\&. If doesn\\*(Aqt, only I<Requires=> is "
"used\\&."
```
- [x] Man page: systemd.mount.5
Issue:    Who is "they"? The type?
```
"If the type is \"overlay\", and \"upperdir=\" or \"workdir=\" are specified "
"as options and they don\\*(Aqt exist, they will be created\\&."
```
- [ ] Man page: systemd-mountfsd.service.8
Issue:    fsck → B<fsck>(8)
```
"The file systems are automatically fsck\\*(Aqed before mounting\\&."
```
- [x] Man page: systemd.netdev.5
Issue:    numbers of devices using it : What ist "it"?
```
"Controls the threshold for broadcast queueing of the macvlan device\\&. "
"Takes the special value \"no\", or an integer in the range "
"0\\&...2147483647\\&. When \"no\" is specified, the broadcast queueing is "
"disabled altogether\\&. When an integer is specified, a multicast address "
"will be queued as broadcast if the number of devices using it is greater "
"than the given value\\&. Defaults to unset, and the kernel default will be "
"used\\&."
```
- [x] Man page: systemd.netdev.5
Issue:    B<systemd-networkd\\&.service> →B<networkd\\&.service>(8)
```
"The Base64 encoded private key for the interface\\&. It can be generated "
"using the B<wg genkey> command (see B<wg>(8))\\&. Specially, if the "
"specified key is prefixed with \"@\", it is interpreted as the name of the "
"credential from which the actual key shall be read\\&.  B<systemd-"
"networkd\\&.service> automatically imports credentials matching \"network\\&."
"wireguard\\&.*\"\\&. For more details on credentials, refer to B<systemd."
"exec>(5)\\&. A private key is mandatory to use WireGuard\\&. If not set, the "
"credential \"network\\&.wireguard\\&.private\\&.I<netdev>\" is used if "
"exists\\&. I\\&.e\\&. for 50-foobar\\&.netdev, \"network\\&.wireguard\\&."
"private\\&.50-foobar\" is tried\\&."
```
- [x] Man page: systemd.net-naming-scheme.7
Issue:    v251 → B<v251>
```
"If we detect that a PCI device associated with a slot is a PCI bridge, we no "
"longer set I<ID_NET_NAME_SLOT>, reverting a change that was introduced in "
"v251\\&."
```
- [x] Man page: systemd.net-naming-scheme.7
Issue:    udev → B<udev>(7)
```
"When creating names for network cards, some naming schemes use data from "
"sysfs populated by the kernel\\&. This means that although a specific naming "
"scheme in udev is picked, the network card\\*(Aqs name can still change when "
"a new kernel version adds a new sysfs attribute\\&. For example if kernel "
"starts setting the B<phys_port_name>, udev will append the "
"\"B<n>I<phys_port_name>\" suffix to the device name\\&."

"This udev property sets a fallback policy for reading a sysfs attribute\\&. "
"If set to B<0> udev will not read any sysfs attribute by default, unless it "
"is explicitly allowlisted, see below\\&. If set to B<1> udev can use any "
"sysfs attribute unless it is explicitly forbidden\\&. The default value is "
"B<1>\\&."

"This udev property explicitly states if udev shall use the specified "
"I<sysfsattr>, when composing the device name\\&."
```
- [x] Man page: systemd.network.5
Issue:    B<systemd-networkd> → B<systemd-networkd>(8)
```
"Note that any network interfaces that have the I<ID_NET_MANAGED_BY=> udev "
"property set will never be matched by any \\&.network files \\(en unless the "
"property\\*(Aqs value is the string \"io\\&.systemd\\&.Network\" \\(en even "
"if the [Match] section would otherwise match\\&. This may be used to exclude "
"specific network interfaces from B<systemd-networkd>\\*(Aqs management, "
"while keeping the [Match] section generic\\&. The I<ID_NET_MANAGED_BY=> "
"property thus declares intended I<ownership> of the device, and permits "
"ensuring that concurrent network management implementations do not compete "
"for management of specific devices\\&."
```
- [x] Man page: systemd.network.5
Issue:    B<systemd-networkd-wait-online> → B<systemd-networkd-wait-online>(8)
```
"Takes a boolean, a minimum operational state (e\\&.g\\&., \"carrier\"), or a "
"range of operational state separated with a colon (e\\&.g\\&., \"degraded:"
"routable\")\\&. Please see B<networkctl>(1)  for possible operational "
"states\\&. When \"yes\", the network is deemed required when determining "
"whether the system is online (including when running B<systemd-networkd-wait-"
"online>)\\&. When \"no\", the network is ignored when determining the online "
"state\\&. When a minimum operational state and an optional maximum "
"operational state are set, B<systemd-networkd-wait-online> deems that the "
"interface is online when the operational state is in the specified range\\&."
```
NAK.
- [x] Man page: systemd.network.5
Issue:    Note\\&. → Note:
```
"Configures IP masquerading for the network interface\\&. If enabled, packets "
"forwarded from the network interface will be appear as coming from the local "
"host\\&. Typically, this should be enabled on the downstream interface of "
"routers\\&. Takes one of \"ipv4\", \"ipv6\", \"both\", or \"no\"\\&. "
"Defaults to \"no\"\\&. Note\\&. Any positive boolean values such as \"yes\" "
"or \"true\" are now deprecated\\&. Please use one of the values above\\&. "
"Specifying \"ipv4\" or \"both\" implies I<IPv4Forwarding=> settings in both "
"\\&.network file for this interface and the global B<networkd.conf>(5)  "
"unless they are explicitly specified\\&. Similarly for I<IPv6Forwarding=> "
"when \"ipv6\" or \"both\" is specified\\&. See I<IPv4Forwarding=>/"
"I<IPv6Forwarding=> in the above for the per-link settings, and B<networkd."
"conf>(5)  for the global settings\\&."
```
- [x] Man page: systemd.network.5
Issue:    forwarding is enabled, → forwarding,
```
"Takes a boolean\\&. Controls IPv6 Router Advertisement (RA) reception "
"support for the interface\\&. If true, RAs are accepted; if false, RAs are "
"ignored\\&. When RAs are accepted, they may trigger the start of the DHCPv6 "
"client if the relevant flags are set in the RA data, or if no routers are "
"found on the link\\&. Defaults to false for bridge devices, when IP "
"forwarding is enabled, I<IPv6SendRA=> or I<KeepMaster=> is enabled\\&. "
"Otherwise, enabled by default\\&. Cannot be enabled on devices aggregated in "
"a bond device or when link-local addressing is disabled\\&."
```
- [x] Man page: systemd.network.5
Issue 1:  \"strict\", mode each → \"strict\" mode, each
Issue 2:  \"loose\", mode each → \"loose\" mode, each
```
"Configure IPv4 Reverse Path Filtering\\&. If enabled, when an IPv4 packet is "
"received, the machine will first check whether the I<source> of the packet "
"would be routed through the interface it came in\\&. If there is no route to "
"the source on that interface, the machine will drop the packet\\&. Takes one "
"of \"no\", \"strict\", or \"loose\"\\&. When \"no\", no source validation "
"will be done\\&. When \"strict\", mode each incoming packet is tested "
"against the FIB and if the incoming interface is not the best reverse path, "
"the packet check will fail\\&. By default failed packets are discarded\\&. "
"When \"loose\", mode each incoming packet\\*(Aqs source address is tested "
"against the FIB\\&. The packet is dropped only if the source address is not "
"reachable via any interface on that router\\&. See \\m[blue]B<RFC "
"3704>\\m[]\\&\\s-2\\u[11]\\d\\s+2\\&. When unset, the kernel\\*(Aqs default "
"will be used\\&."
```
- [x] Man page: systemd.network.5
Issue:    B<systemd-networkd-wait-online> → B<systemd-networkd-wait-online>(8)
```
"With this enabled, to make the interface enter the \"configured\" state, "
"which is required to make B<systemd-networkd-wait-online> work properly for "
"the interface, all dynamic address configuration mechanisms like I<DHCP=> "
"and I<IPv6AcceptRA=> (which is enabled by default in most cases) need to be "
"disabled\\&. Also, I<DuplicateAddressDetection=> (which is enabled by "
"default for IPv4 link-local addresses and all IPv6 addresses) needs to be "
"disabled for all static address configurations\\&. Otherwise, without "
"carrier, the interface will be stuck in the \"configuring\" state, and "
"B<systemd-networkd-wait-online> for the interface will timeout\\&. Also, it "
"is recommended to set I<RequiredForOnline=no-carrier> to make B<systemd-"
"networkd-wait-online> work for the interface\\&."
```
NAK.
- [x] Man page: systemd.network.5
Issue:    B<systemd-networkd> → B<systemd-networkd>(8)
```
"When an interface is configured with IP addresses, the addresses, subnetwork "
"masks or interface index will be appended to the NFT sets\\&. The "
"information will be removed when the interface is deconfigured\\&.  "
"B<systemd-networkd> only inserts elements to (or removes from) the sets, so "
"the related NFT rules, tables and sets must be prepared elsewhere in "
"advance\\&. Failures to manage the sets will be ignored\\&."
```
NAK.
- [x] Man page: systemd.network.5
Issue:    A boolean → Takes a boolean
```
"A boolean\\&. Specifies whether the rule is to direct lookups to the tables "
"associated with level 3 master devices (also known as Virtual Routing and "
"Forwarding or VRF devices)\\&. For further details see \\m[blue]B<Virtual "
"Routing and Forwarding (VRF)>\\m[]\\&\\s-2\\u[23]\\d\\s+2\\&. Defaults to "
"false\\&."
msgstr ""
"Akzeptiert einen logischen Wert\\&. Gibt an, ob die Regel lautet, dass "
"Nachschlagen in den Tabellen, die Stufe-3-Master-Geräten zugeordnet sind, "
"erfolgen sollen (auch als virtuelles Routing und Weiterleiten von VRF-"
"Geräten bekannt)\\&. Zu weiteren Details siehe \\m[blue]B<Virtuelles Routen "
"und Weiterleiten (VRF)>\\m[]\\&\\s-2\\u[23]\\d\\s+2\\&. Standardmäßig "
"falsch\\&."
```

- [x] Man page: systemd.network.5
Issue:    in the above → above
```
"Note that this filters only DHCP offers, so the filtering might not work "
"when I<RapidCommit=> is enabled\\&. See also I<RapidCommit=> in the above\\&."
```
- [x] Man page: systemd.network.5
Issue:    and configures routes → and routes
```
"When true (the default), Redirect message sent by the current first-hop "
"router will be accepted, and configures routes to redirected nodes will be "
"configured\\&."
```
- [x] Man page: systemd.network.5
Issue:    systemd-networkd\\&.service → B<systemd-networkd\\&.service>(8)
```
"Takes a boolean\\&. When true, the DHCP server will load and save leases in "
"the persistent storage\\&. When false, the DHCP server will neither load nor "
"save leases in the persistent storage\\&. Hence, bound leases will be lost "
"when the interface is reconfigured e\\&.g\\&. by B<networkctl reconfigure>, "
"or systemd-networkd\\&.service is restarted\\&. That may cause address "
"conflict on the network\\&. So, please take an extra care when disable this "
"setting\\&. When unspecified, the value specified in the same setting in "
"B<networkd.conf>(5), which defaults to \"yes\", will be used\\&."
```
- [x] Man page: systemd.network.5
Issue:    which indicate → indicate  (or remove "that")
```
"Takes a boolean\\&. Specifies that IPv6 router advertisements which indicate "
"to hosts that the router acts as a Home Agent and includes a Home Agent "
"option\\&. Defaults to false\\&. See \\m[blue]B<RFC "
"6275>\\m[]\\&\\s-2\\u[13]\\d\\s+2 for further details\\&."
```
- Man page: systemd-network-generator.service.8
Issue:    systemd-networkd\\&.service → B<systemd-networkd\\&.service>(8)

"Note that the resulting files are created world-readable, it\\*(Aqs hence "
"recommended to not include secrets in these credentials, but supply them via "
"separate credentials directly to systemd-networkd\\&.service\\&."
--
- [x] Man page: systemd-network-generator.service.8
Issue:    systemd-network-generator\\&.service → B<systemd-network-generator\\&.service>
```
"Note that by default the systemd-network-generator\\&.service unit file is "
"set up to inherit the these credentials from the service manager\\&."
```
- [x] Man page: systemd-nspawn.1
Issue:    Missing full stop
```
"When running in unprivileged mode, some needed functionality is provided via "
"B<systemd-mountfsd.service>(8)  and B<systemd-nsresourced.service>(8)"
```
- [x] Man page: systemd-nspawn.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
NAK
- [x] Man page: systemd.pcrlock.5
Issue:    B<systemd-pcrlock>(1) → B<systemd-pcrlock>(8)
```
"*\\&.pcrlock files define expected TPM2 PCR measurements of components "
"involved in the boot process\\&.  B<systemd-pcrlock>(1)  uses such pcrlock "
"files to analyze and predict TPM2 PCR measurements\\&. The pcrlock files are "
"JSON arrays that follow a subset of the \\m[blue]B<TCG Canonical Event Log "
"Format (CEL-JSON)>\\m[]\\&\\s-2\\u[1]\\d\\s+2 specification\\&. Specifically "
"the \"recnum\", \"content\", and \"content_type\" record fields are not used "
"and ignored if present\\&. Each pcrlock file defines one set of expected, "
"ordered PCR measurements of a specific component of the boot\\&."
```
- [x] Man page: systemd.pcrlock.5
Issue:    B<systemd-pcrlock> → B<systemd-pcrlock>(8)
```
"Use the various B<lock-*> commands of B<systemd-pcrlock> to automatically "
"generate suitable pcrlock files for various types of resources\\&."
```
NAK
- [x] Man page: systemd-pcrlock.8
Issue:    The word "format" is already in the cited name, remove "format" at the end of the sentence?
```
"This reads the combined TPM2 event log and writes it to STDOUT in "
"\\m[blue]B<TCG Canonical Event Log Format (CEL-"
"JSON)>\\m[]\\&\\s-2\\u[1]\\d\\s+2 format\\&."
```
- [x] Man page: systemd-pcrlock.8
Issue:    B<systemd-stub> → B<systemd-stub>(7)
```
"Generates/removes a \\&.pcrlock file based on a kernel initrd cpio "
"archive\\&. This is useful for predicting measurements the Linux kernel "
"makes to PCR 9 (\"kernel-initrd\")\\&. Do not use for B<systemd-stub> UKIs, "
"as the initrd is combined dynamically from various sources and hence does "
"not take a single input, like this command\\&."
```
- [x] Man page: systemd-pcrlock.8
Issue:    overrides → configures
```
"Takes a file system path as argument\\&. If specified overrides where to "
"write pcrlock policy metadata to\\&. If not specified defaults to /var/lib/"
"systemd/pcrlock\\&.json\\&."
```
- [x] Man page: systemd-pcrphase.service.8
Issue:    systemd-pcrphase\\&.service → B<systemd-pcrphase\\&.service>
```
"\"shutdown\" \\(em when the system shutdown begins\\&. It acts as barrier "
"between the time the system is fully up and running and where it is about to "
"shut down\\&. (This extension happens when the systemd-pcrphase\\&.service "
"service is stopped\\&.)"
```
- [x] Man page: systemd-portabled.service.8
Issue:    In systemd-portabled.service(8): Portable Services Documentation
```
"Portable Services"
```
- [x] Man page: systemd-poweroff.service.8
Issue:    systemd-poweroff\\&.service → B<systemd-poweroff.service>
```
"systemd-poweroff\\&.service is a system service that is pulled in by "
"poweroff\\&.target and is responsible for the actual system power-off "
"operation\\&. Similarly, systemd-halt\\&.service is pulled in by halt\\&."
"target, systemd-reboot\\&.service by reboot\\&.target and systemd-kexec\\&."
"service by kexec\\&.target to execute the respective actions\\&."

"Note that systemd-poweroff\\&.service (and the related units) should never "
"be executed directly\\&. Instead, trigger system shutdown with a command "
"such as \"systemctl poweroff\"\\&."
```
- [x] Man page: systemd-poweroff.service.8
Issue:    systemd-shutdown → B<systemd-shutdown>
```
"Shortly before executing the actual system power-off/halt/reboot/kexec "
"systemd-shutdown will run all executables in /usr/lib/systemd/system-"
"shutdown/ and pass one arguments to them: either \"poweroff\", \"halt\", "
"\"reboot\", or \"kexec\", depending on the chosen action\\&. All executables "
"in this directory are executed in parallel, and execution of the action is "
"not continued before all executables finished\\&. Note that these "
"executables are run I<after> all services have been shut down, and after "
"most mounts have been unmounted (the root file system as well as /run/ and "
"various API file systems are still around though)\\&. This means any "
"programs dropped into this directory must be prepared to run in such a "
"limited execution environment and not rely on external services or "
"hierarchies such as /var/ to be around (or writable)\\&."
```
- [x] Man page: systemd-poweroff.service.8
Issue 1:  systemd-shutdown → B<systemd-shutdown>
Issue 2:  have been detached → have been unmounted (?)
```
"Shortly before executing the actual system power-off/halt/reboot/kexec "
"systemd-shutdown will run all executables in /usr/lib/systemd/system-"
"shutdown/ and pass one arguments to them: either \"poweroff\", \"halt\", "
"\"reboot\", or \"kexec\", depending on the chosen action\\&. All executables "
"in this directory are executed in parallel, and execution of the action is "
"not continued before all executables finished\\&. Note that these "
"executables are run I<after> all services have been shut down, and after "
"most mounts have been detached (the root file system as well as /run/ and "
"various API file systems are still around though)\\&. This means any "
"programs dropped into this directory must be prepared to run in such a "
"limited execution environment and not rely on external services or "
"hierarchies such as /var/ to be around (or writable)\\&."
```
- [ ] Man page: systemd-repart.8
Issue:    fstab → B<fstab>(5)
```
"Specifies a path where to write fstab entries for the mountpoints configured "
"with B<MountPoint=> in the root directory specified with B<--copy-source=> "
"or B<--root=> or in the host\\*(Aqs root directory if neither is "
"specified\\&. Disabled by default\\&."
```
- [x] Man page: systemd-repart.8
Issue:    PKCS11\\&. → PKCS11:
```
"The following creates a system extension DDI (sysext) for an /usr/foo update "
"and signs it with a hardware token via PKCS11\\&."
```
- [x] Man page: systemd-repart.8
Issue:    In systemd-portabled.service(8): Portable Services Documentation
```
"portable service"
```
- [x] Man page: systemd-resolved.service.8
Issue:    systemd-resolved\\&.service → B<systemd-resolved\\&.service>
```
"The nss-dns resolver maintains little state between subsequent DNS queries, "
"and for each query always talks to the first listed DNS server from /etc/"
"resolv\\&.conf first, and on failure continues with the next until reaching "
"the end of the list which is when the query fails\\&. The resolver in "
"systemd-resolved\\&.service however maintains state, and will continuously "
"talk to the same server for all queries on a particular lookup scope until "
"some form of error is seen at which point it switches to the next, and then "
"continuously stays with it for all queries on the scope until the next "
"failure, and so on, eventually returning to the first configured server\\&. "
"This is done to optimize lookup times, in particular given that the resolver "
"typically must first probe server feature sets when talking to a server, "
"which is time consuming\\&. This different behaviour implies that listed DNS "
"servers per lookup scope must be equivalent in the zones they serve, so that "
"sending a query to one of them will yield the same results as sending it to "
"another configured DNS server\\&."
```
- [x] Man page: systemd.resource-control.5
Issue:    may defined → to be defined
```
"Configures restrictions on the ability of unit processes to invoke "
"B<bind>(2)  on a socket\\&. Both allow and deny rules may defined that "
"restrict which addresses a socket may be bound to\\&."
```
- [x] Man page: systemd.resource-control.5
Issue:    B<systemd-coredump> → B<systemd-coredump>(8)
```
"When B<systemd-coredump> is handling a coredump for a process from a "
"container, if the container\\*(Aqs leader process is a descendant of a "
"cgroup with I<CoredumpReceive=yes> and I<Delegate=yes>, then B<systemd-"
"coredump> will attempt to forward the coredump to B<systemd-coredump> within "
"the container\\&."
```
- [x] Man page: systemd-rfkill.service.8
Issue:    kernel command line → B<kernel-command-line>(7)
```
"systemd-rfkill understands the following kernel command line parameter:"
```
NAK.
- [x] Man page: systemd.service.5
Issue:    Missing full stop at the end
```
"An argument solely consisting of \";\" must be escaped, i\\&.e\\&. specified "
"as \"\\e;\""
```
- [x] Man page: systemd-soft-reboot.service.8
Issue:    sd-bus → B<sd-bus>(3)
```
"If the unit publishes a service over D-Bus, the connection needs to be re-"
"established after soft-reboot as the D-Bus broker will be stopped and then "
"started again\\&. When using the sd-bus library this can be achieved by "
"adapting the following example\\&."
```
- [x] Man page: systemd-soft-reboot.service.8
Issue:    systemd-soft-reboot\\&.service → B<systemd-soft-reboot.service>
```
"Note that systemd-soft-reboot\\&.service (and related units) should never be "
"executed directly\\&. Instead, trigger system shutdown with a command such "
"as \"systemctl soft-reboot\"\\&."
```
- [x] Man page: systemd-soft-reboot.service.8
Issue:   In systemd-portabled.service(8): Portable Services Documentation
```
"Portable Services"
```
- [ ] Man page: systemd-ssh-generator.8
Issue:    B<ssh> → B<ssh>(1)
```
"See B<systemd-ssh-proxy>(1)  for details on how to connect to these sockets "
"via the B<ssh> client\\&."
```
- [x] Man page: systemd-ssh-generator.8
Issue:    systemd-ssh-generator → B<systemd-ssh-generator>
```
"systemd-ssh-generator implements B<systemd.generator>(7)\\&."

"systemd-ssh-generator understands the following B<kernel-command-line>(7)  "
"parameters:"
```
- [x] Man page: systemd-ssh-generator.8
Issue:    systemd → B<systemd>(1)
```
"This option configures an additional socket to bind SSH to\\&. It may be "
"used multiple times to bind multiple sockets\\&. The syntax should follow "
"the one of I<ListenStream=>, see B<systemd.socket>(5)  for details\\&. This "
"functionality supports all socket families systemd supports, including "
"B<AF_INET> and B<AF_INET6>\\&."
```
- [x] Man page: systemd-ssh-proxy.1
Issue:    of VM guest → or VM guest?
```
"This tool is supposed to be used together with B<systemd-ssh-generator>(8)  "
"which when run inside a VM or container will bind SSH to suitable "
"addresses\\&.  B<systemd-ssh-generator> is supposed to run in the container "
"of VM guest, and B<systemd-ssh-proxy> is run on the host, in order to "
"connect to the container or VM guest\\&."
```
- [x] Man page: systemd-storagetm.service.8
Issue:    systemd-storagetm\\&.service → B<systemd-storagetm\\&.service>
```
"systemd-storagetm\\&.service is a service that exposes all local block "
"devices as NVMe-TCP mass storage devices\\&. Its primary use-case is to be "
"invoked by the storage-target-mode\\&.target unit that can be booted into\\&."
```
- [x] Man page: systemd-stub.7
Issue:    these system extension images → these configuration extension images
```
"Similarly, files I<foo>\\&.efi\\&.extra\\&.d/*\\&.confext\\&.raw are packed "
"up in a B<cpio> archive and placed in the /\\&.extra/confext/ directory in "
"the initrd file hierarchy\\&. This is supposed to be used to pass additional "
"configuration extension images to the initrd\\&. See B<systemd-confext>(8)  "
"for details on configuration extension images\\&. The generated B<cpio> "
"archive containing these system extension images is measured into TPM PCR 12 "
"(if a TPM is present)\\&."
```
- [x] Man page: systemd-sysext.8
Issue:    earlier: → earlier
```
"A simple mechanism for version compatibility is enforced: a system extension "
"image must carry a /usr/lib/extension-release\\&.d/extension-release\\&."
"I<NAME> file, which must match its image name, that is compared with the "
"host os-release file: the contained I<ID=> fields have to match unless "
"\"_any\" is set for the extension\\&. If the extension I<ID=> is not "
"\"_any\", the I<SYSEXT_LEVEL=> field (if defined) has to match\\&. If the "
"latter is not defined, the I<VERSION_ID=> field has to match instead\\&. If "
"the extension defines the I<ARCHITECTURE=> field and the value is not "
"\"_any\" it has to match the kernel\\*(Aqs architecture reported by "
"B<uname>(2)  but the used architecture identifiers are the same as for "
"I<ConditionArchitecture=> described in B<systemd.unit>(5)\\&.  "
"I<EXTENSION_RELOAD_MANAGER=> can be set to 1 if the extension requires a "
"service manager reload after application of the extension\\&. Note that for "
"the reasons mentioned earlier: \\m[blue]B<Portable "
"Services>\\m[]\\&\\s-2\\u[2]\\d\\s+2 remain the recommended way to ship "
"system services\\&. System extensions should not ship a /usr/lib/os-release "
"file (as that would be merged into the host /usr/ tree, overriding the host "
"OS version data, which is not desirable)\\&. The extension-release file "
"follows the same format and semantics, and carries the same content, as the "
"os-release file of the OS, but it describes the resources carried in the "
"extension image\\&."
```
- [x] Man page: systemd-sysext.8
Issue:    systemd-confext → B<systemd-confext>
```
"With systemd-confext one can perform runtime reconfiguration of OS "
"services\\&. Sometimes, there is a need to swap certain configuration "
"parameter values or restart only a specific service without deployment of "
"new code or a complete OS deployment\\&. In other words, we want to be able "
"to tie the most frequently configured options to runtime updateable flags "
"that can be changed without a system reboot\\&. This will help reduce "
"servicing times when there is a need for changing the OS configuration\\&. "
"It also provides a reliable tool for managing configuration because all old "
"configuration files disappear when the systemd-confext image is removed\\&."
```
- [x] Man page: systemd-sysext.8
Issue:    In systemd-portabled.service(8): Portable Services Documentation
```
"Portable Services"
```
- [x] Man page: systemd-sysext.8
Issue 1:  Is $name correct? Not name?
Issue 2:  escape hatch of xattr ???
Issue 3:  xattr → B<xattr>(7)
```
"Again, just like sysext images, the confext images will contain a /etc/"
"extension-release\\&.d/extension-release\\&.I<$name> file, which must match "
"the image name (with the usual escape hatch of xattr), and again with "
"content being one or more of I<ID=>, I<VERSION_ID=>, and "
"I<CONFEXT_LEVEL>\\&. Confext images will then be checked and matched against "
"the base OS layer\\&."
```
- [x] Man page: systemd.time.7
Issue:    What does &c after Tokyo stand for?
```
"The timezone defaults to the current timezone if not specified "
"explicitly\\&. It may be given after a space, like above, in which case it "
"can be: \"UTC\", an entry in the installed IANA timezone database (\"CET\", "
"\"Asia/Tokyo\", &c\\&.; complete list obtainable with \"timedatectl list-"
"timezones\" (see B<timedatectl>(1))), or \"\\(+-I<05>\", \"\\(+-"
"I<05>I<30>\", \"\\(+-I<05>:I<30>\", \"Z\"\\&."
```
- [x] Man page: systemd-tmpfiles.8
Issue:    packaged → package
```
"The primary usecase for this option is to automatically remove files and "
"directories that originally have been created on behalf of an installed "
"packaged at package removal time\\&."
```
- [x] Man page: systemd-tmpfiles.8
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
NAK
- [x] Man page: systemd-tpm2-generator.8
Issue:    systemd-tpm2-generator → B<systemd-tpm2-generator>
```
"systemd-tpm2-generator is a generator that adds a I<Wants=> dependency from "
"sysinit\\&.target to tpm2\\&.target when it detects that the firmware "
"discovered a TPM2 device but the OS kernel so far did not\\&.  tpm2\\&."
"target is supposed to act as synchronization point for all services that "
"require TPM2 device access\\&. See B<systemd.special>(7)  for details\\&."

"systemd-tpm2-generator implements B<systemd.generator>(7)\\&."
```
- [x] Man page: systemd.unit.5
Issue:    then this the → then this
```
"For units that start processes (such as service units), lists one or more "
"other units whose network and/or temporary file namespace to join\\&. If "
"this is specified on a unit (say, a\\&.service has I<JoinsNamespaceOf=b\\&."
"service>), then the inverse dependency (I<JoinsNamespaceOf=a\\&.service> for "
"b\\&.service) is implied\\&. This only applies to unit types which support "
"the I<PrivateNetwork=>, I<NetworkNamespacePath=>, I<PrivateIPC=>, "
"I<IPCNamespacePath=>, and I<PrivateTmp=> directives (see B<systemd.exec>(5)  "
"for details)\\&. If a unit that has this setting set is started, its "
"processes will see the same /tmp/, /var/tmp/, IPC namespace and network "
"namespace as one listed unit that is started\\&. If multiple listed units "
"are already started and these do not share their namespace, then it is not "
"defined which namespace is joined\\&. Note that this setting only has an "
"effect if I<PrivateNetwork=>/I<NetworkNamespacePath=>, I<PrivateIPC=>/"
"I<IPCNamespacePath=> and/or I<PrivateTmp=> is enabled for both the unit that "
"joins the namespace and the unit whose namespace is joined\\&."
```
outdated
- [x] Man page: systemd-vmspawn.1
Issue:    systemd-vmspawn → B<systemd-vmspawn>
```
"Note: on Ubuntu/Debian derivatives systemd-vmspawn requires the user to be "
"in the \"kvm\" group to use the VSOCK options\\&."
```
- [ ] Man page: systemd-vmspawn.1
Issue:    B<sshd> → B<sshd>(8)
```
"By default \"ed25519\" keys are generated, however \"rsa\" keys may also be "
"useful if the VM has a particularly old version of B<sshd>\\&."
```
- [x] Man page: systemd-vmspawn.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
NAK
- [x] Man page: systemd-vpick.1
Issue:    it specified → its specified
```
"If the specified path does not reference a \"\\&.v/\" path (i\\&.e\\&. "
"neither the final component ends in \"\\&.v\", nor the penultimate does or "
"the final one does contain a triple underscore) it specified path is written "
"unmodified to standard output\\&."
```
- [x] Man page: timedatectl.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
NAK
- [x] Man page: tmpfiles.d.5
Issue:    B<systemd-tmpfiles> → B<systemd-tmpfiles>(8)
```
"By passing this line to QEMU, the public key of the current user will be "
"encoded in base64, added to a tmpfiles\\&.d line that tells B<systemd-"
"tmpfiles> to decode it into /root/\\&.ssh/authorized_keys, encode that line "
"itself in base64 and pass it as a Credential that will be picked up by "
"systemd from SMBIOS on boot\\&."
```
- [x] Man page: ukify.1
Issue:    last words: to the file named I<PATH> or to a file in the I<PATH>?
```
"For the B<inspect> verb, the second syntax is used\\&. The section I<NAME> "
"will be inspected (if found)\\&. If the second argument is \"text\", the "
"contents will be printed\\&. If the third argument is given, the contents "
"will be saved to file I<PATH>\\&."
```
- [x] Man page: ukify.1
Issue:    Why is this a hard coded link to a man page? Should be B<systemd-stub>(1) instead
```
"SBAT metadata associated with the UKI or addon\\&. SBAT policies are useful "
"to revoke whole groups of UKIs or addons with a single, static policy update "
"that does not take space in DBX/MOKX\\&. If not specified manually, a "
"default metadata entry consisting of \"uki,1,UKI,uki,1,https://uapi-group\\&."
"org/specifications/specs/unified_kernel_image/\" for UKIs and \"uki-addon,1,"
"UKI Addon,addon,1,https://www\\&.freedesktop\\&.org/software/systemd/man/"
"latest/systemd-stub\\&.html\" for addons will be used, to ensure it is "
"always possible to revoke them\\&. For more information on SBAT see "
"\\m[blue]B<Shim documentation>\\m[]\\&\\s-2\\u[2]\\d\\s+2\\&."
```
The link is included in the metadata.
- [x] Man page: userdbctl.1
Issue 1:  B<less> → B<less>(1)
Issue 2:  systemd → B<systemd>(1)
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
NAK

**Comment by hmartink** [comment]
Hello Yu,
thanks for your review of the translators comments. I have a few replies, maybe you can take them into account.

1) The references to other man pages serve a valuable tool for readers to get more information. Some HTML renders even make them available as hyperlink. Thus I think it is sensible to include them (I noticed that you tend to reject them, especially if they go to tools outside systemd).

2) systemd(1) is quite general, it explains the concepts as well.

3) I hope I got your systematic right, some have a cross and are not taken, some don't have a cross. Those which you are not accepting are marked as such in our repository, so they won't be reported again.

4) For  pam_systemd_loadkey.8.po: The man page pam(7) does not use the term "authtok". The term "authtok" seems to be an abbreviation. Thus I propose to spell it out (at least once)

5)  The first issue in "run0.1" leaves me still wondering …

6) In case of questions, a short rationale would have helped, some you did not comment on.

7)  systemd-rfkill.service.8 - You have this in other man pages, it was just for consistency within systemd.

8) chroot(8) is existing

9) Answer to ukify(1): The link is in the body of the text, not in the metadata as usual. (It is for the sbat reference in the same paragraph).

Thanks for handling this vast body of documentation!

**Comment by yuwata** [comment]
@hmartink Thank you for the report and comments.
>     1. The references to other man pages serve a valuable tool for readers to get more information. Some HTML renders even make them available as hyperlink. Thus I think it is sensible to include them (I noticed that you tend to reject them, especially if they go to tools outside systemd).

Of course, agree with that in general. But, I do not want to add the same citation multiple times in the same section or paragraph. Though, there is no strict rule in our side, but at least in my opinion.

>     2. systemd(1) is quite general, it explains the concepts as well.

Is this comment for the following?
```
"Note that setting the regular I<$LESS> environment variable has no effect "
"for B<less> invocations by systemd tools\\&."

"Note that setting the regular I<$LESSCHARSET> environment variable has no "
"effect for B<less> invocations by systemd tools\\&."
```
Again, systemd(1) explains mainly about the service manager, that is, init system. But, here, `systemd tools` means many binaries provided by systemd project. Especially, here for management tools like systemctl, networkctl, resolvectl, udevadm, and so on. So, systemd(1) is not a good citation here. Of course, the term `systemd tools` is slightly ambiguous, but there is no good citation point to be added here. Most closest one is systemd.index(7), but it also listed all library functions...

>     3. I hope I got your systematic right, some have a cross and are not taken, some don't have a cross. Those which you are not accepting are marked as such in our repository, so they won't be reported again.

Unfortunately, not systematic. So, requests are welcome. But, again, I do not want to add citation too many times, e.g. systemd.network(5) mentions systemd-networkd(8), networkctl(1), systemd-networkd-wait-online(8) at many places, but I only add citation at only few places. Though, the decision when cited and when not is still not systematic.

>     4. For  pam_systemd_loadkey.8.po: The man page pam(7) does not use the term "authtok". The term "authtok" seems to be an abbreviation. Thus I propose to spell it out (at least once)

OK. I will add some suitable citation. Maybe, pam_get_authtok(3) or similar.

>     5. The first issue in "run0.1" leaves me still wondering …

That means run0 is a symbolic link of systemd-run. When invoked as run0, then it behaves as run0, otherwise, it behaves as systemd-run.

>     6. In case of questions, a short rationale would have helped, some you did not comment on.

I thought all checked without 'NAK' are addressed. If I mistakenly checked without any comments, then please ask.

>     7. systemd-rfkill.service.8 - You have this in other man pages, it was just for consistency within systemd.

Actually we have kernel-command-line(7), but it is not suitable here. Maybe we should list more options supported by the service.

>     8. chroot(8) is existing

In my laptop, there is only chroot(1) and chroot(2). And chroot(8) seems to be for FreeBSD. And, here, `chroot environment` should be cited. But, chroot(1) is for the command, chroot(2) is for API. I think both do not fit here.

>     9. Answer to ukify(1): The link is in the body of the text, not in the metadata as usual. (It is for the sbat reference in the same paragraph).

Not familiar, will recheck.

> Thanks for handling this vast body of documentation!

Thank you for the feedback!

**Comment by yuwata** [comment]
-> #35365

**Comment by hmartink** [comment]
Thanks. I'll see where a NAK I missed and mark those as "possible not accept" or similar in the next round, so you can distinguish them.

And chroot(8) is in coreutils, maybe this is not installed on your laptop.

**Comment by yuwata** [comment]
> And chroot(8) is in coreutils, maybe this is not installed on your laptop.

Of course it is installed...
```
$ sudo dnf list coreutils*
Last metadata expiration check: 0:01:36 ago on Thu Nov 28 03:28:58 2024.
Installed Packages
coreutils.x86_64                          9.4-9.fc40                    @updates
coreutils-common.x86_64                   9.4-9.fc40                    @updates
Available Packages
coreutils-single.x86_64                   9.4-9.fc40                    updates 

$ rpm -qa --filesbypkg coreutils* | grep chroot
coreutils-common          /usr/share/man/man1/chroot.1.gz
coreutils                 /usr/sbin/chroot
```
Could you give me a link to chroot(8) ?
Only I found for linux is https://manpages.debian.org/testing/coreutils/chroot.8.en.html . And the upstream provides only https://github.com/coreutils/coreutils/blob/master/man/chroot.x  . I guess debian installs chroot(1) as chroot(8) for taking compatibility to BSD's chroot(8) ?
Either way, chroot(8) is not suitable there, I think.
At least, chroot(2) is better, if I need to chose one of them.


Anyway. Thank you for all your feedback, that's much appreciated.
Further request is of course always welcome.

**Comment by hmartink** [comment]
Yes, I looked at the build scripts of the Debian package, it kind of renames chroot(1) into chroot(8). I wasn't aware of this.
